### PR TITLE
More cleanups: remove sequence API, tests no feebump and batching paths

### PR DESCRIPTION
**Pull request by darosior**

### CODE REVIEW

**Comment by darosior** [comment]
Rebased.

**Comment by darosior** [comment]
Moar rebase

File: src/transactions.rs [hunk 1]
```diff
@@ -997,12 +1042,13 @@ mod tests {
             &vec![feebump_xpriv],
             None,
             SigHashType::All,
-        );
+        )
+        .unwrap();
         emergency_tx.finalize().unwrap();
 
         // Create but don't sign the unvaulting transaction until all revaulting transactions
         // are
```
Comment (JSwambo) [inline src/transactions.rs:1050]: nit: "are signed."

**Review by JSwambo — APPROVED** [review]
Ack 238cb6ec19dfd6550712b8a932beb980237e5a22

Good thinking to split out the fee-bump inputs!

**Pull request merged**